### DISCUSSION TITLE
Welcome to AI0x0.com Discussions!

### DISCUSSION THREAD

**Discussion by mushan0x0** [mushan0x0/AI0x0.com#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by laiyerui** [comment]
often can not connect the baichat

**Comment by loverHaoCheng** [comment]
The latest version does not support MacOS Montetey 12.4!

**Comment by loverHaoCheng** [comment]
After starting up in macOS, the mouse often drifts uncontrollably to the upper left corner, hoping to fix this bug

**Comment by gyk185yyds** [comment]
最新版本win10联网查询没有跳转网页的连接，上一个版本支持
